packaging/package_info.rst

```diff
@@ -0,0 +1,190 @@
+Package identifiers and information
+====================================
+
+Packages provide two types of information for consumers. The first one would be the package
+identifiers (ID), which is a SHA-1 hash of the package configuration (settings, options, requirements),
+that allows consumers to reuse an existing package without building it again from sources.
+
+The other type of information would be C/C++ build information, as include paths, library names, or
+compile flags
```

Comment:
Well, it is printed in the ``conan info`` command, is useful information for consumers of the package, to know which binary they are actually using for a given configuration. 

But, yes, I agree that the title/description could be improved.